static/js/settings_bots.js

```diff
@@ -510,20 +507,8 @@ export function set_up() {
                         cache: false,
                         processData: false,
                         contentType: false,
-                        success(data) {
+                        success() {
                             avatar_widget.clear();
-                            if (data.avatar_url) {
-                                // Note that the avatar_url won't actually change on the backend
-                                // when the user had a previous uploaded avatar.  Only the content
-                                // changes, so we version it to get an uncached copy.
-                                image_version += 1;
-                                $image
-                                    .find("img")
-                                    .attr(
-                                        "src",
-                                        data.avatar_url + "&v=" + image_version.toString(),
```

Comment:
Though we will be removing this code, but I added this commit in case anyone goes through commit history and finds that while reusing the code we do not take these lines into account and ignore them. Thus, I added this commit to correctly document that why we can remove this.